extract/extract/extractors/zalando.py

```diff
@@ -136,9 +136,19 @@ def get_sustainability(
         List[str]: Ordered `list` of found sustainability labels
     """
 
-    if cluster := beautiful_soup.find("div", attrs={"data-testid": "cluster-certificates"}):
-        labels = cluster.find_all(attrs={"data-testid": "certificate__title"})
+    data = json.loads(beautiful_soup.find("script", {"type": "application/json",
+                                                     "class": "re-1-13"}).get_text())
+    labels = []
+    for key, item in data.get("graphqlCache", {}).items():
+        for entry in item.get("data", {}).get("product", {}).get("attributeSuperClusters", [{}]):
+            if entry.get("id", "") == "sustainability":
+                for cluster in entry.get("clusters", [{}]):
+                    if cluster.get("sustainabilityClusterKind", "") == "certificates":
+                        for attribute in cluster.get("attributes", [{}]):
+                            labels.append(attribute.get("label", "").replace("%25", "%"))
```

Comment:
I just realized that the `%25` is a percent encoded representation of `%`, so an encoding should be more appropriate here. But this will be a task for next week ;)